<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsApplication1Class</class>
 <widget class="QMainWindow" name="QtWidgetsApplication1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1425</width>
    <height>911</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsApplication1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>1001</width>
      <height>801</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>结构线条工具</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>直线</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>选择和移动</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>三点曲线</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>四点曲线</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>291</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>结构纸样工具</string>
    </property>
    <widget class="QToolButton" name="toolButton_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>水平垂直线</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>纸样用例1</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_10">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>200</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>顺滑曲线</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>水平垂直切口</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>水平垂直翻转</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>290</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>水平垂直校正</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>纸样用例2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>纸样用例3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>选择和移动</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1425</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionOpen_File_O"/>
    <addaction name="actionQuit_Q"/>
    <addaction name="actionQuit_Q_2"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionUndo_Ctrl_Z"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp_Manual_Ctrl_H"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionopen">
   <property name="text">
    <string>New File  (Ctrl+N)</string>
   </property>
  </action>
  <action name="actionOpen_File_O">
   <property name="text">
    <string>Open File (Ctrl+O)</string>
   </property>
  </action>
  <action name="actionQuit_Q">
   <property name="text">
    <string>Save (Ctrl+S)</string>
   </property>
  </action>
  <action name="actionQuit_Q_2">
   <property name="text">
    <string>Quit (Ctrl+Q)</string>
   </property>
  </action>
  <action name="actionUndo_Ctrl_Z">
   <property name="text">
    <string>delete (Del)</string>
   </property>
  </action>
  <action name="actionHelp_Manual_Ctrl_H">
   <property name="text">
    <string>Help Manual (Ctrl+H)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsApplication1.qrc"/>
 </resources>
 <connections/>
</ui>
